### DISCUSSION TITLE
My university stoped issuing student mail

### DISCUSSION THREAD

**Discussion by Bialri** [community/community#49716] [GitHub Education]
I am a student of Saint Petersburg state university of economics (UNECON). My university stoped providing of academic email, but it is necessary for registration in the program. I have proof in the form of a student id and a certificate from my university. Can I register without email?

**Comment by ghost** [comment]
Hi there! It's understandable that you're worried about not being able to register for the program without an academic email. However, there may be other options available to you.

Firstly, you can try reaching out to the program administrator or admissions office to explain your situation and ask if there are alternative methods for registration that don't require an academic email. They may be able to provide you with a workaround or an alternative email address that you can use.

If that doesn't work, you can also try using a personal email address for registration. While it's not ideal, it may be a viable option as long as the program doesn't require an academic email specifically.

Lastly, if neither of these options work, you can also consider reaching out to your university's IT department to see if they have any plans to reinstate academic email services or if there are any other alternatives they can offer you.

I hope this helps you find a solution and successfully register for the program! Let me know if you have any further questions or if there's anything else I can assist you with.

**Reply by Bialri** [reply]
Hello, I had already submitted a request to the IT department of my university and the answer was negative. Also, I can't find any contacts for the administrator of the program. Could you please give me a link or something to help with this.

**Reply by ghost** [reply]
> aint Petersburg state university of economics (UNECON

I'm sorry to hear that you've already reached out to your university's IT department and they were unable to help you with your academic email issue. Unfortunately, I don't have a link or contact information for the administrator of the program you're trying to register for, but you might want to try searching online or contacting the program's support team directly for assistance.

If all else fails, you can consider using a personal email address for registration, as I suggested earlier. I hope this helps, and I wish you the best of luck in finding a solution.

**Reply by Bialri** [reply]
Thank you, I already wrote to support, I was told to write to this section. I will try again. This reminds me of ping pong. 😔

**Comment by samus-aran** [comment]
Hi @Bialri 👋 

You can use your personal email, but depending on the institution they may request that an academic email is required. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Hope this helps. Let us know how you get on 👍

**Reply by Bialri** [reply]
<img width="728" alt="Снимок экрана 2023-03-11 в 08 01 27" src="https://user-images.githubusercontent.com/62287690/224466094-5d032964-4b18-43cc-9ab8-da009d10e019.png">
Hello, I can't continue registration, there is no place where I can upload my student ID or certificate. I'm fully stuck with it. I have already sent request to support and I got nothing.
